models/Comment.php

```diff
@@ -17,23 +17,59 @@ public function get_comments_blog(){
         
         return $results;
     }
-
+    
+    // Fonction permettant de vérifier que les données envoyées pat l'utilisateur sont bonnes.
+    function form_comment_verification(){
+        if(filter_has_var(INPUT_POST, 'submit')){
+            $name = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'name', FILTER_SANITIZE_STRING)));
+            $email = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'email', FILTER_SANITIZE_STRING)));
+            $comment = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'comment', FILTER_SANITIZE_STRING)));
+            $errors = [];
+
+            if(empty($name) || empty($email) || empty($comment)){
+                $errors['empty'] = "Tous les champs n'ont pas été remplis";
+            }else{
+                if(!filter_var($email, FILTER_VALIDATE_EMAIL)){
+                    $errors['email'] = "L'adresse email n'est pas valide";
+                }
+            }
+
+            if(!empty($errors)){
+            ?>
+                <div class="card red">
+                    <div class="card-content white-text">
+                        <?php
+                            foreach($errors as $error){
+                                echo $error;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)